csm_web/frontend/src/components/Section.js

```diff
@@ -108,7 +108,9 @@ export function SectionSpacetime({ spacetime: { location, time }, override, chil
 SectionSpacetime.propTypes = {
   spacetime: SPACETIME_SHAPE.isRequired,
   override: PropTypes.shape({ spacetime: SPACETIME_SHAPE.isRequired, date: PropTypes.string.isRequired }),
-  children: PropTypes.node
+  children: PropTypes.node,
+  manySpacetimes: PropTypes.bool.isRequired,
+  index: PropTypes.number.isRequired
```

Comment:
maybe add a comment for posterity clarifying that this is 0-indexed but the frontend displays 1-indexed